taranis/__main__.py

```diff
@@ -166,6 +166,12 @@ def taranis_cli(verbose, log_file):
     default=False,
     help="Remove no CDS alleles from the schema.",
 )
+@click.option(
+    "--output-allele-annot/--no-output-allele-annot",
+    required=False,
+    default=True,
+    help="get extension annotation for all alleles in locus",
```

Comment:
```suggestion
    help="output prokka/allele annotation for alleles in locus",
```
this is for saving or not prokka annotation? Or for the big txt file with allele annotation and quality? Well one of those options i propose here!

Reply:
This parameter is for getting full annotation, which include, prokka annotation, DNA sequence, Protein sequence and quality for each allele in the locus.
The other file you are talking about, it is not implemented yet, as I would like to do it when implementing the allele calling feature, to know the optimal way to fetch the information

Reply:
ok, change the help description so it is clearer, output prokka annotation and quality info for each allele gene, or something like that